artillery-tests/data/erc20.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity ^0.6.0;
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract GGXToken is ERC20 {
-    constructor(uint256 initialSupply) public ERC20("Golden Gate Token", "GGX") {
+    constructor(uint256 initialSupply) public ERC20("GGX Token", "GGX") {
```

Comment:
@akorchyn Not GGXT?

Reply:
It's just for tests mock. We actually don't use solidity except tests